internal/testregistry/registry.go

```diff
@@ -0,0 +1,68 @@
+package testregistry
```

Comment:
I don't think we need this, we already use the Docker distribution, please see https://github.com/fluxcd/source-controller/blob/main/controllers/suite_test.go#L120